packages/tracing/src/index.ts

```diff
@@ -1,10 +1,10 @@
-import { BrowserTracing } from './browser';
 import { addExtensionMethods } from './hubextensions';
-import * as TracingIntegrations from './integrations';
-
-const Integrations = { ...TracingIntegrations, BrowserTracing };
+import * as Integrations from './integrationsbundle';
 
 export { Integrations };
+
```

Comment:
```suggestion
// this is already exported as part of `Integrations` (and for the moment will remain so for backwards compatibility), but that interferes with treeshaking, so we also export it separately here
```

Reply:
Thanks for this, expanded upon this comment and added.